src/xbps/troubleshooting/common-errors.md

```diff
@@ -0,0 +1,32 @@
+# Common errors
+
+## Errors while updating or installing packages
+
+If there are any errors while updating or installing a new package, make sure
+that you are using an updated version of the remote repository index. Running
+[xbps-install(1)](https://man.voidlinux.org/xbps-install.1) with the `-S` option
+will guarantee that.
+
+### shlib errors
+
+If you get an error with the format
+
+```
+libllvm8-8.0.1_2: broken, unresolvable shlib `libffi.so.6'
+```
+
+It is likely that orphan packages which have already been removed from the Void
+repos are still installed in your system. This can be solved by running
+[xbps-remove(1)](https://man.voidlinux.org/xbps-remove.1) with the `-o` option,
+which removes orphan packages.
+
+If you get an error message saying that
+
+```
+Transaction aborted due to unresolved shlibs
+```
+
+The repositories are in the staging state, which can happen due to large builds.
```

Comment:
"the repositories" - no capital, since this is a continuation of the sentence before the example.